protocol/avdl/chat1/common.avdl

```diff
@@ -460,12 +460,20 @@
     @mpackkey("b") @jsonkey("b")
     union { null, gregor1.UID } botUID;
 
+    // DEPRECATED
     // [V1, V2]: Missing
     // [V3, V4]: Specifies which stellar transaction ID is associated with the
     // message, if any, to do server-side validation of the transaction
     // notification.
     @mpackkey("t") @jsonkey("t")
     union { null, stellar1.TransactionID} txID;
+
+    // [V1, V2]: Missing
+    // [V3, V4]: Specifies which stellar transaction IDs are associated with
+    // the message, if any, to do server-side validation of the transaction
+    // notification.
+    @mpackkey("ts") @jsonkey("ts")
+    union { null, array<stellar1.TransactionID>} txIDs;
```

Comment:
Can you error out if both fields are present when boxing/unboxing?

Do we access the `txID` field anywhere in the code base? Maybe there should be a helper which returns `[]stellar1.TransactionID`, pulling the data from whichever field is populated